features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,21 @@
-# put step definitions in this file
\ No newline at end of file
+Given(/^I am on the home page$/) do
+  visit '/'
+end
+
+When(/^I visit the name page for "([^"]*)"$/) do |arg1|
+  visit '/name/Amber'
+end
+
+Given(/^The following users exists$/) do |table|
+  table.hashes.each do |user|
+    @user2 = User.create(name: user[:name])
+  end
+end
+
+Then(/^I should create a new user with name "([^"]*)"$/) do |arg1|
+  expect(@user2.name).to eq "#{arg1}"
+end
```

Comment:
Good - here you are using the `arg1` variable. But you should give it a name that makes sense, like `name`.